mempool/v1/mempool.go

```diff
@@ -178,6 +178,15 @@ func (txmp *TxMempool) CheckTx(tx types.Tx, cb func(*abci.Response), txInfo memp
 	// During the initial phase of CheckTx, we do not need to modify any state.
 	// A transaction will not actually be added to the mempool until it survives
 	// a call to the ABCI CheckTx method and size constraint checks.
+
+	// check if the transaction contains blobs. If so, replace the transaction
+	// with the wrapped one
```

Comment:
we replacing the transaction with the wrapped transaction, but in order to do that we have to unwrap it

have any ideas for naming wrapped and unwrapped? I only went with it because that's what we use for malleated transactions. It isn't the worst name in the world, but it kinda feels like we're abusing it a bit

Reply:
I think we can resolve this conversation without applying my suggestion above. I was confused on the behavior of `UnwrapBlobTx` but commented elsewhere for that.